<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <author/>
    <comment/>
    <exportmacro/>
    <class>ShareDTClientWin</class>
    <widget class="QWidget" name="ShareDTClientWin">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>ShareDTClientWin</string>
        </property>

        <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>600</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QLabel" name="imageLabel">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <property name="frameShape">
                            <enum>QFrame::Box</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Plain</enum>
                        </property>
                        <property name="lineWidth">
                            <number>3</number>
                        </property>
                        <property name="text">
                            <string>TextLabel</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>


        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>24</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuEdit">
                <property name="title">
                    <string>Edit</string>
                </property>
                <addaction name="actionNew_connection"/>
            </widget>
            <widget class="QMenu" name="menuWindow">
                <property name="title">
                    <string>Window</string>
                </property>
                <addaction name="actionAdjust_to_original_size"/>
            </widget>
            <widget class="QMenu" name="menuHelp">
                <property name="title">
                    <string>Help</string>
                </property>
                <addaction name="actionShow_help"/>
            </widget>
            <addaction name="menuEdit"/>
            <addaction name="menuWindow"/>
            <addaction name="menuHelp"/>
        </widget>
        <action name="actionnew">
            <property name="text">
                <string>new</string>
            </property>
        </action>
        <action name="actionNew_connection">
            <property name="text">
                <string>New connection</string>
            </property>
        </action>
        <action name="actionAdjust_to_original_size">
            <property name="text">
                <string>Adjust to original size</string>
            </property>
        </action>
        <action name="actionShow_help">
            <property name="text">
                <string>Show Help</string>
            </property>
        </action>


    </widget>
    <pixmapfunction/>
    <connections/>
</ui>